### PR TITLE
Majors minors

### PR DESCRIPTION
**Pull request by katrina433**
### Summary 

This PR includes the layout of the majors/minors page

- The title component with a placeholder for the quiz button
- The dashboard component with hard-coded lists of majors, minors, and graduate studies
- The apply tags component with checkboxes (doesn't actually apply the filter)

lF1EPpoR1F

### Test Plan 

Can change the lists in `src/areas-of-study/AreasOfStudy.tsx` to check if the corresponding section gets updated. 

Can change the tags lists in `src/areas-of-study/dashboard/ApplyTags.tsx` to check if the Apply Tags component gets updated.

### CODE REVIEW

File: design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { 
+  ApplyTagsContainer
+} from '../AreasOfStudyStyles';
+
+const ApplyTags = () => {
+  const designAreaTags = ['Art + Design', 'Digital Design', 'Industrial Design', 'Policy Design', 'Spatial Design']
+  const schools = ['AAP', 'Agriculture and Life Sciences', 'Arts and Sciences', 'Engineering', 'Human Ecology', 'ILR', 'SC Johnson School of Buisness']
```
Comment (jessfeng) [inline design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx]: Business spell check!

File: design-at-cornell/src/constants/pages.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ type Page = {
 
 const pages: Page[] = [
   { name: 'Explore Courses', url: '/explore-courses' },
-  { name: 'Majors & Minors', url: '/majors-&-minors' },
+  { name: 'Majors & Minors', url: '/areas-of-study' },
```
Comment (jessfeng) [inline design-at-cornell/src/constants/pages.ts:8]: Remember to also change this for the footer!

File: design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  const checkboxes = (tags: string[]) => (
+    <form>
+      {tags.map(tag => (
+        <div>
+          <input key={tag} type="checkbox" id={tag} />
+          <label htmlFor={tag}>{tag}</label>
+        </div>
+      ))}
+    </form>
+ )
```
Comment (jessfeng) [inline design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx:19]: Is it possible to align the text and checkboxes a bit more? It's a bit shifted down, but also no biggie if we can't.
Screen Shot 2021-04-01 at 11 34 48 PM

**Review by jessfeng — COMMENTED** [review]
Excellent work ✨ It looks really neat and clean, which is amazing! I left a couple nitpicky comments, but overall it looks great :) One thing to keep in mind is probably adjusting the padding/margin for the right side (next to the tags), since sometimes it can look a bit squished, but responsiveness is something we can fine-tune in later sprints.

File: design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx (hunk 3 above)
Comment (katrina433) [reply design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx:19]: I added vertical alignment to the checkboxes, so they should theoretically be aligned with the text. But they still don't seem to be 100% aligned (though much better than before) and I don't know what to fix that :(

Comment (jessfeng) [reply design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx:19]: Oh okay, no worries! That should be good for now then :)

**Review by jessfeng — APPROVED** [review]
Looks good to me! Just remember to resolve the conflicts before merging into master--good work!

**Pull request merged**